### DISCUSSION TITLE
Multiple ldap user federation

### DISCUSSION THREAD

**Discussion by andys448** [keycloak/keycloak#16142] [Q&A]
Hi All!

We have a two domains a.com / b.com:

Both have kerberos authorization configured with their own keytab file. Priorities (1 / 2) are configured for a.com and b.com and their simultaneous use is impossible, when both are active, only the one with a lower priority works and people from the b.com domain cannot enter the application using kerberos. Thus, if both ldaps are active, then keycloak does not check for a user in the b.com domain.

Thanks for any help!

**Comment by abramovks** [comment]
I didn't resolve with kerberos/ntlm. Work only first domain in list.

I use SAML server (Identity providers menu) that connected to more than 10 domain controllers  for internal AD users
